src/docs/08-workflow-troubleshooting/03-retries.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Retries
+permalink: /docs/workflow-troubleshooting/retries
+---
+
+# Retries
+
+Cadence has a retry feature where a retry policy can be configured so that an activity or a workflow can be retried when it fails or times out.
```

Comment:
```suggestion
Cadence has a retry feature where a retry policy can be configured so that an activity or a workflow will be retried when it fails or times out.
```